miniwin/src/internal/d3drmrenderer_opengl15.h

```diff
@@ -10,6 +10,12 @@
 
 DEFINE_GUID(OPENGL15_GUID, 0x682656F3, 0x0000, 0x0000, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x03);
 
+struct GLTextureCacheEntry {
+	IDirect3DRMTexture* texture;
+	Uint32 version;
+	GLuint glTextureId;
+};
```

Comment:
I'd put this struct inside the `OpenGL15Renderer` as a subclass.

Reply:
Ok, I look forward to seeing how so I can learn :)